plugins/basic/python/tk_photoshopcc_basic/__init__.py

```diff
@@ -8,4 +8,5 @@
 # agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
 # not expressly granted therein are reserved by Shotgun Software Inc.
 
-from .plugin_bootstrap import bootstrap_toolkit
\ No newline at end of file
+from .classic_init import toolkit_classic_bootstrap
+from .plugin_init import toolkit_plugin_bootstrap
```

Comment:
i personally think it's nice with the encapuslation - have one python file which contains the classic bootstrap and one that contains the plugin one (and it's various helper methods).

Having said that, there used to be more code, then i ended up removing a bunch, so the need to split things into two files is not as strong anymore. I'll see if i can merge them back together..